src/explore-education-statistics-admin/src/pages/publication/components/PublicationForm.tsx

```diff
@@ -25,7 +25,12 @@ const errorMappings = [
   mapFieldErrors<FormValues>({
     target: 'title',
     messages: {
-      SlugNotUnique: 'Choose a unique title',
+      PublicationSlugNotUnique: 'Choose a unique title',
+      PublicationSlugUsedByRedirect: 'Choose a previously unused title',
+      MethodologySlugNotUnique:
+        "As this publication's owned methodology inherits this publication's title, this action changes the owned methodology. Choose a unique methodology title",
```

Comment:
Just wondering if these messages need any vetting from Cam?  I'd be tempted to try something a little less wordy, like "This title also changed the methodology title to one used by another methodology. Choose a unique methodology title".  Same for the one below.  Not a big deal if we don't change it though!

Reply:
FYI I spoke with @cjrace and we confirmed wording should read:

"This title also changed the methodology title to one used by an existing methodology. Choose a unique methodology title"

Reply:
I changed that and another message, so I ran them all by Cam again and he gave it the :+1:

Reply:
Nice, ta!